requirements.txt

```diff
@@ -20,4 +20,4 @@ pytest-cov==2.6.0
 grpcio-tools==1.15.0
 cryptography==2.3.1
 ldap3==2.5.1
-tornado==4.5.3
+tornado==5.1.1
```

Comment:
Have you checked this for listener compatibility with rethinkdb at runtime? I have to down-rev it after encountering issues with an added arg.

Reply:
that was after running pipreqs. I'll change and rebase